recipes/crashpad/all/conanfile.py

```diff
@@ -1,10 +1,14 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conans import AutoToolsBuildEnvironment, tools
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
 from contextlib import contextmanager
+import conan.tools.files as tools_files
+import conan.tools.scm as tools_scm
 import os
+import sys
 import textwrap
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.46.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
Because conan.errors is available since 1.50.0

Reply:
Also, replace `build_requires` by `tools_requires` please. If that same error persist, I'll check other dependencies.

Reply:
Will do, thanks!